README.md

```diff
@@ -21,7 +21,7 @@ You will need the following SDKs, tools, and apps installed to work through the
 You will also need a local AEM author setup:
 
 - AEM 6.5 author instance running on `:4502`
-    - Ideally, with the latest [Service Pack](https://docs.adobe.com/content/help/en/experience-manager-65/release-notes/service-pack/sp-release-notes.html) installed
+  - Ideally, with the latest [Service Pack](https://docs.adobe.com/content/help/en/experience-manager-65/release-notes/service-pack/sp-release-notes.html) installed
```

Comment:
Already mentioned in the main README.  Do we need to mention it again?
https://github.com/adobe/aem-search-experiments